package.json

```diff
@@ -20,6 +21,9 @@
     "jest": "^27.0.6"
   },
   "dependencies": {
-    "eslint-config-hackreactor": "github:reactorcore/eslint-config-hackreactor"
+    "eslint-config-hackreactor": "github:reactorcore/eslint-config-hackreactor",
```

Comment:
I did not know this was a package you could install!

Reply:
yes you can :) I found out during the fec project. https://github.com/reactorcore/eslint-config-hackreactor